src/bundles/ApplicationsAdmin/components/AppDiff/AppDiff.css

```diff
@@ -0,0 +1,18 @@
+tr.row {
+	border-top-color: black;
+	border-top-style: solid;
+	border-top-width: 2px;
```

Comment:
This can be shortened to `border-top: 2px solid black;`

Reply:
fixed